### DISCUSSION TITLE
How to contribute to the docs written in README, etc. is outdated

### DISCUSSION THREAD

**Discussion by reminjp** [chakra-ui/chakra-ui#9281] [Q&A] [answered]
Hello. When I was interested in contributing to the docs, I found these instructions in the README:

> Feel like contributing? That's awesome! We have a
> [contributing guide](./CONTRIBUTING.md) to help guide you.
> 
> Our docsite lives in a
> [separate repo](https://github.com/chakra-ui/chakra-ui-docs). If you're
> interested in contributing to the documentation, check out the
> [docsite contribution guide](https://github.com/chakra-ui/chakra-ui-docs/blob/main/CONTRIBUTING.md).

However, the "separate repo" has already been archived. Perhaps it's too early to try contributing to the v3 docs?

**Comment by veloware** [comment]
likewise, the Select docs aren't correct for the react-hook-form example and need updated, and I would like to know the process

**Comment by segunadebayo** [comment, marked as answer]
Duplicate of https://github.com/chakra-ui/chakra-ui/discussions/9419

The old `chakra-ui-docs` repo is archived. Docs live in this repo. https://chakra-ui.com/docs/get-started/contributing
